docs/user/exercises/quiz.rst

```diff
@@ -1,2 +1,440 @@
+
+            - Short question title: Provide a short title (Mandatory).
+
+            - Points: Assign the value points for this question.
+
+            - Scoring type:
+
+                - All or nothing
+                - Proportional with Penalty
+                - Proportional without Penalty
+
+            - Present answer options in random order.
+
+            - Single Choice Mode: When there is just one correct option. This disables the Scoring type.
+
+            - Delete icon: To delete the current question.
+
+            - Edit View: Enables the text editor to write the quiz statement and its options, hints and explanations.
+
+            - Edit bar: When the edit view is enabled, the format bar provides:
+
+                - Style to the statement text
+
+                - Correct Options [correct]
+
+                - Incorrect Options [wrong]
+
+                - Explanations [exp]
+
+                - Hints [hint]
+
+            - Text editor: The quiz statement can be developed with options, hints and explanations.
+
+            - Preview View: Enables the student view.
+
+        - **Add Drag-And-Drop Question**
+
+            This kind of question is composed of a problem statement, a background image, and drag and drop options.
+
+            .. figure:: quiz/drag-and-drop-question.png
+                :align: center
+
+            - Short question title: Provide a short title.
+
+            - Points: Assign the value points for this question.
+
+            - Scoring type:
+
+                - All or nothing
+
+                - Proportional with Penalty
+
+                - Proportional without Penalty
+
+            - Present Drag Items in Random order.
+
+            - Delete icon: To delete the current question.
+
+            - Edit View: Enables the text editor to write the question statement with explanations and hints.
+
+            - Edit bar: When the edit view is enabled, the format bar provides:
+
+                - Style to the statement text
+
+                - Explanations [exp]
+
+                - Hints [hint]
+
+            - Text editor: The quiz statement can be developed with hints and explanations.
+
+            - Upload Background: To select and upload the background from the PC files to drag and drop the options over it.
+
+            - Add Drag Items:
+
+                - Text items: Type the options.
+
+                - Image items: Can be uploaded from the PC files.
+
+            - Preview View: Enables the student view.
+
+        - **Add Short-Answer Question**
+
+            This kind of question is composed of a statement and spots to fill them out by typing the answers.
+
+            .. figure:: quiz/short-answer-question.png
+                :align: center
+
+            - Short question title: Provide a short title.
+
+            - Points: Assign the value points for this question.
+
+            - Scoring type:
+
+                - All or nothing
```

Comment:
```suggestion
                - All or Nothing
```
(capitalization as in the screenshot)

Reply:
The option in Artemis is established as "All or Nothing"